mimic/util.py

```diff
@@ -20,4 +20,5 @@ def verify_text(clean_text):
     """Verify that the text is suitable for ML."""
     if len(clean_text.split()) < 20000:
         # TODO should we make a custom error class?
+        print(len(clean_text.split()))
```

Comment:
This should probably be a call to `logging`, not `print`.

Reply:
Thanks for the catch; that was just for testing